### ISSUE TITLE
MultiVI with Poisson/NB likelihood for ATAC?

### ISSUE THREAD

**Issue by vitkl** [scverse/scvi-tools#2257]
Hi @martinkim0 and all

Similarly to https://github.com/scverse/scvi-tools/pull/2249 by @lauradmartens would be great to have a MultiVI version with Poisson or even NB likelihood for ATAC data. How hard do you @martinkim0 think it is to add such option to MultiVI?

**Comment by canergen** [comment]
Hi @vitkl I'm linking @marianogabitto here as he developed multiVI and was also working on a different implementation of PoissonPeakVI. What's your opinion on this request?

**Comment by lauradmartens** [comment]
Yes, I also think it should be relatively straightforward and also remember that @marianogabitto already had an implementation

**Comment by vitkl** [comment]
Thanks for getting back to me about this! If you @marianogabitto have an implementation of this - would be great if you could share it or add it to scvi-tools (if makes sense for @canergen)

**Comment by canergen** [comment]
Happy to review the PR.

**Comment by vitkl** [comment]
I don't understand the implementations well enough to easily try implementing this - but I can do a PR if I see the example implementation from @marianogabitto

**Comment by canergen** [comment]
I'm sorry for the confusion that was a comment to @marianogabitto to assign me the PR if he creates one.

**Comment by Juliendly** [comment]
Hi all, I am very interested in this implementation and I'm curious to know if there is any updates on it?
Thanks!

**Comment by canergen** [comment]
@lauradmartens congrats on the new manuscript: https://www.biorxiv.org/content/10.1101/2024.09.19.613754v2. Any plan to continue with the Poisson MultiVI branch.
